### ISSUE TITLE
App doesn't start after reinstalling

### ISSUE THREAD

**Issue by julien4215** [lichess-org/mobile#736]
To reproduce :

1. Uninstall the app from the PlayStore
2. Reinstall the app from the PlayStore
3. Launch the app

Here are the logs I get for the error :
```
I/flutter (20933): SEVERE: [App] could not load app dependencies; PlatformException(Exception encountered, read, javax.crypto.BadPaddingException: error:1e000065:Cipher functions:OPENSSL_internal:BAD_DECRYPT
I/flutter (20933): 	at com.android.org.conscrypt.NativeCrypto.EVP_CipherFinal_ex(Native Method)
I/flutter (20933): 	at com.android.org.conscrypt.OpenSSLEvpCipher.doFinalInternal(OpenSSLEvpCipher.java:152)
I/flutter (20933): 	at com.android.org.conscrypt.OpenSSLCipher.engineDoFinal(OpenSSLCipher.java:374)
I/flutter (20933): 	at javax.crypto.Cipher.doFinal(Cipher.java:2055)
I/flutter (20933): 	at s6.h.b(Unknown Source:35)
I/flutter (20933): 	at r6.a.d(Unknown Source:11)
I/flutter (20933): 	at r6.a.n(Unknown Source:17)
I/flutter (20933): 	at r6.e$b.run(Unknown Source:243)
I/flutter (20933): 	at android.os.Handler.handleCallback(Handler.java:938)
I/flutter (20933): 	at android.os.Handler.dispatchMessage(Handler.java:99)
I/flutter (20933): 	at android.os.Looper.loop(Looper.java:223)
I/flutter (20933): 	at android.os.HandlerThread.run(HandlerThread.java:67)
I/flutter (20933): , null)
```

The app seems to work fine after emptying the storage of the app.

**Comment by veloce** [comment]
Do you uninstall and reinstall through play store?

**Comment by julien4215** [comment]
> Do you uninstall and reinstall through play store? 

Yes I updated the issue

**Comment by 0xj0hn** [comment]
Which version are you running? @julien4215

**Comment by julien4215** [comment]
Android 11

**Comment by 0xj0hn** [comment]
> Android 11

The version of the app, I meant.

**Comment by julien4215** [comment]
latest 0.8.16

**Comment by veloce** [comment]
Got the confirmation it is fixed by #737
